### ISSUE TITLE
CSS bug in Explorer Pane for Github sandbox

### ISSUE THREAD

**Issue by boonyasukd** [codesandbox/codesandbox-client#2185]
# 🐛 bug report

## Description of the problem
In my github-imported sandbox, the Explorer pane has a display problem when the tree is expanded. Using a mouse wheel on Explorer pane seems to be offset by a script that automatically put the top of the file tree back in place, so it's really hard to mitigate. I have to switch to the other menu then come back to Explorer pane in order to force a CSS reflow/recalculation.

## How has this issue affected you?
It's hard to access files in the tree when the tree is expanded.

![buggy_css_reflow](https://user-images.githubusercontent.com/8109816/61457368-5ea98c80-a992-11e9-88c6-259e9b106671.png)

### Your Environment

| Software         | Name/Version |
| ---------------- | ------------ |
| Сodesandbox  | PROD-1563302711-343dbe14e   |
| Browser     |  Google Chrome Version 75.0.3770.142 (Official Build) (64-bit)   |
| Operating System | Win 10 (64-bit) |

**Comment by CompuIves** [comment]
Wooow, I wonder when this regression appeared. Going to take a look now!

**Comment by CompuIves** [comment]
Interesting, I can only reproduce it in production. Wondering what's happening here...

**Comment by boonyasukd** [comment]
@CompuIves

I tried to play around a bit again, and it seems that the problem is on the collapsible **Dependencies**, not **File** like I previously reported. I tried to click it to show/hide npm packages, but it doesn't seem to work at all: the npm packages seem to be in "expanded" state all the time. But again, if I switch to any other menu on the left (e.g., "Github" integration menu) and come back, it starts to work as it should again.

Since I don't see this on my regular (non-Github) sandboxes, I guess this might occur because of the height (mis-)calculation when there's an ad below **Files** and **Dependencies** (in the above screenshot, it's that **CodeFund** block).
